install-scripts/install-debian.sh

```diff
@@ -0,0 +1,40 @@
+# This installation has been tested on Debian 11 XFCE
```

Comment:
Recommendations on some lines to add at the very beggining of the script (they are useful in almost any bash script you may develop):
```shell
#!/usr/bin/env bash
set -ex
```

* `#!/usr/bin/env bash` tells the operating system to run this script with bash, this is a common convention to be able to run the script directly by doing `./install-debian.sh` (the file must have execute permissions for this)
* the `set` command  is used to modify the different attributes and parameters of the shell environment, in this case:
  * the `-e` flag makes the script end/fail after a command inside the script fails (otherwise after a failure the following commands would be executed anyways)
  * the `-x` flag displays the command being executed, it's useful to debug the script as it's running

Reply:
#!/usr/bin/env bash tells the operating system to run this script with bash

Does it execute this line despite it being commented out? :O